README.md

```diff
@@ -24,27 +24,27 @@ LevelDOWN
 Introduction
 ----------------------------
 
-LevelDOWN was extracted from [LevelUP](https://github.com/level/levelup) and now serves as a stand-alone binding for LevelDB.
+This module was originally part of [`levelup`](https://github.com/level/levelup) but was later extracted and now serves as a stand-alone binding for LevelDB.
```

Comment:
I rewrote this to get a capital letter. Not really sure if we should keep this or not, do people ending up on leveldown really care about the lore and why it was extracted? If so, should that be the first sentence?

Reply:
That said, I'm fine with keeping it as well.

Reply:
> Not really sure if we should keep this or not, do people ending up on leveldown really care about the lore and why it was extracted?

Nah. I think it's fine to remove the first sentence. (The TOC isn't that long here so it's close enough to the short description)